junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/Constants.java

```diff
@@ -93,6 +167,15 @@ public final class Constants {
 	 */
 	public static final String DEACTIVATE_ALL_CONDITIONS_PATTERN = ClassNamePatternFilterUtils.ALL_PATTERN;
 
+	/**
+	 * A blank pattern used for class name filtering: {@value}
+	 *
+	 * <p>This constant is used to represent an empty or blank pattern in class name filtering operations.
+	 *
+	 * @see ClassNamePatternFilterUtils#BLANK
+	 */
+	public static final String BLANK = ClassNamePatternFilterUtils.BLANK;
+
```

Comment:
It seems weird to me to introduce a constant for an empty string here. Let's keep it internal and remove it from here, ok?